From ca09da570efe365d47c727b728749dd2635ec6b7 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v2-3.4-ca09da570ef-20220520T072122Z-avarab@gmail.com>
In-Reply-To: <cover-v2-0.4-00000000000-20220520T072122Z-avarab@gmail.com>
References: <cover-0.3-00000000000-20220406T104134Z-avarab@gmail.com>
	<cover-v2-0.4-00000000000-20220520T072122Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 20 May 2022 09:15:59 +0200
Subject: [PATCH v2 3/4] cocci: remove env_array -> env migration
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Remove the coccinelle rule renaming "env_array" to "env". In the
preceding commit we carried out this rename based on this rule, anyone
with out-of-tree patches still referring to "env" will now get a
compilation error.

We could keep the rule to help those developers change their code by
running "make coccicheck", but as the solution is obvious in this case
let's not leave this stale rule in place.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 contrib/coccinelle/run_command.cocci | 11 -----------
 1 file changed, 11 deletions(-)
 delete mode 100644 contrib/coccinelle/run_command.cocci

diff --git a/contrib/coccinelle/run_command.cocci b/contrib/coccinelle/run_command.cocci
deleted file mode 100644
index 709fdfcf720..00000000000
--- a/contrib/coccinelle/run_command.cocci
+++ /dev/null
@@ -1,11 +0,0 @@
-@@
-struct child_process E;
-@@
-- E.env_array
-+ E.env
-
-@@
-struct child_process *E;
-@@
-- E->env_array
-+ E->env
-- 
2.36.1.957.g2c13267e09b

